[DOCS] Update case priv docs (#1443)
elastic/security-docs: docs/getting-started/sec-app-requirements.asciidoc

To use {es-sec}, you must have at least:

* `Read` privilege for the `Security` feature in the {kib} space (see

> {kibana-ref}/xpack-spaces.html[Spaces]). This grants you `Read` access to all features in {es-sec} except cases. For more information about the minimum privileges needed to use cases, refer to <<case-permissions, Case prerequisites>>.

Reviewer: Optional suggestion -- more concise, a little more conversational?
```suggestion
{kibana-ref}/xpack-spaces.html[Spaces]). This grants you `Read` access to all features in {es-sec} except cases. For more information about the minimum privileges needed to use cases, see <<case-permissions, Case prerequisites>>.
```

Author: I always bounce between these two because I do think "refer to" is overly formal and a bit impersonal. On the other hand, I feel as though "see" is ableist and exclusionary language. What about "go to" or "read"?

Reviewer: Valid point about ableist language. We use "see" _a lot_ throughout the docs (4 times on this very page!), but IMO that doesn't necessarily mean we have to keep using it. I still think "refer to" and other alternatives sound dry or formal, so maybe consider a different way to avoid that kind of construction altogether?
```suggestion
{kibana-ref}/xpack-spaces.html[Spaces]). This grants the user `Read` access to all features in {es-sec} except cases. Additional <<case-permissions, minimum privileges>> are needed to use cases.
```

Author: Really like this alternative. Thanks!

Reviewer 2: Thanks for bringing this up, you two. I notice that the Kibana team uses "refer to..." but it does sound a bit formal IMO. I'll address it at the next UX leads meeting. @nastasha-solomon don't want to overwrite Joe's suggestion, but fix the `es-sec` attribute to `elastic-sec`.
